domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/response/HowMuchOwed.java

```diff
@@ -0,0 +1,56 @@
+package uk.gov.hmcts.cmc.domain.models.response;
+
+import org.hibernate.validator.constraints.NotBlank;
+import uk.gov.hmcts.cmc.domain.constraints.Money;
+import java.math.BigDecimal;
+import java.util.Objects;
+import javax.validation.constraints.DecimalMin;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+
+public class HowMuchOwed {
+
+    @NotNull
+    @Money
+    @DecimalMin(value = "0.01")
+    private final BigDecimal amount;
+
+    @NotBlank()
+    @Size(max = 255, message = "Explanation should not be longer than {max} characters")
```

Comment:
Please remove custom message and make validation between frontend and backend aligned. Frontend allows 99k characters.